gradle/libs.versions.toml

```diff
@@ -20,44 +20,44 @@ androidx-compose = "1.1.1"
 androidx-composeThemeAdapter = "1.1.5"
 androidx-datastore = "1.0.0"
 androidx-espresso = "3.4.0"
-androidx-lifecycle-common = "2.4.0"
-androidx-lifecycle-compose = "2.4.0"
-androidx-other-constraintLayout = "2.1.1"
+androidx-lifecycle-common = "2.4.1"
+androidx-lifecycle-compose = "2.4.1"
+androidx-other-constraintLayout = "2.1.3"
 androidx-other-appCompat = "1.3.1"
-androidx-other-media = "1.4.2"
+androidx-other-media = "1.6.0"
 androidx-other-dynamicAnimation = "1.0.0"
-androidx-other-preference = "1.1.1"
+androidx-other-preference = "1.2.0"
 androidx-other-core = "1.6.0"
 androidx-other-viewPager2 = "1.0.0"
 androidx-other-swipeRefreshLayout = "1.1.0"
 androidx-other-recyclerView = "1.2.1"
-androidx-other-fragment = "1.3.6"
-androidx-other-browser = "1.3.0"
-androidx-other-workManager = "2.7.0"
-androidx-other-startup = "1.1.0"
+androidx-other-fragment = "1.4.1"
+androidx-other-browser = "1.4.0"
+androidx-other-workManager = "2.7.1"
+androidx-other-startup = "1.1.1"
 androidx-other-transition = "1.4.1"
 androidx-other-activityCompose = "1.4.0"
 androidx-test-core = "1.4.0"
 androidx-test-junit = "1.1.3"
-arrowKt = "1.0.1"
+arrowKt = "1.1.2"
 assertK = "0.25"
-adyen = "4.2.0"
+adyen = "4.6.0"
 coil = "1.4.0"
 coilTransformations = "1.0.0"
 combineTuple = "1.2.1"
 concatAdapterExtension = "1.2.1"
 coreLibraryDesugaring = "1.1.5"
-coroutines = "1.6.0"
+coroutines = "1.6.1"
 firebase = "28.4.2"
 flexbox = "3.0.0"
 fragmentViewBindingDelegate = "1.0.0"
 hAnalytics = "v0.256.0"
-insetter = "0.6.0"
-jsonTest = "20210307"
+insetter = "0.6.1"
+jsonTest = "20220320"
 kaspresso = "1.4.1"
-koin = "3.1.2"
-leakCanary = "2.7"
-lottie = "4.2.0"
+koin = "3.2.0"
+leakCanary = "2.9.1"
+lottie = "5.1.1"
```

Comment:
Where do we use this actually? I think there's definitely some room of removing some deps here and there if it's stuff we no longer use.

Reply:
We use `lottie` to display the audio recorder-action in the chat. Which to be fair is no longer used, I guess we can probably cut that out 😄

Reply:
I'm going to remove `lottie` as a separate PR, I think, in order to manage the scope of this PR.